colors/material.vim

```diff
@@ -57,6 +57,7 @@ endfun
 
 " Vim Editor
 call s:HL('ColorColumn', '', s:gray3, '')
+call s:HL('Conceal', '', s:gray1, '')
```

Comment:
Awesome, thank you. I'd definitely like to make this change, but I'm wondering what you think of [matching the color of the conceal characters to that of comments](https://github.com/kaicataldo/material.vim/pull/19/files#diff-693a3fb1845dd2c150138d65ec2fe8d6R102) to indicate that they don't have syntactic meaning in the language?

```suggestion
call s:HL('Conceal', s:gray4, '', 'italic')
```

Reply:
At least in this case, with markdown+latex, Conceal is used to process the syntax and produce an output - so it should blend in with normal text. Gray text is hard to read. You can also have a look at:

https://github.com/lifepillar/vim-solarized8/blob/40e77bf0730e86a59538c5472cf30bc21a3c5d87/colors/solarized8.vim#L91

https://github.com/lifepillar/vim-solarized8/blob/40e77bf0730e86a59538c5472cf30bc21a3c5d87/colors/solarized8.vim#L160

Reply:
Got it. That makes sense to me. That being said, it looks like this change sets the background color rather than the foreground color (though your screenshot seems to show the opposite). Is that intentional?

Reply:
I am changing the background color and **that is intentional**. I have not touched the foreground color, so it is what it is. If you don't trust me, please try it yourself locally. Install https://github.com/vim-pandoc/vim-pandoc-syntax and open any markdown file!